operator/src/main/java/oracle/kubernetes/operator/helpers/HttpClientPool.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
I think this class is unnecessary because you don't actually pool HttpClient.  Instead, you can share a single instance.

Reply:
ok, I'll remove HttpClientPool class and HttpClientFactory classes. I will introduce a static

Reply:
Yes, looks good.

The pool you copied this from is also no longer needed, but removing that is technical debt. When the underlying HTTP client implementation was OkHttp 2.x, the pooling was needed, but OkHttp 3.x has its own internal pooling.

Reply:
ok, got it. I merged change based on above comments in ba303f0.